src/components/Counter.js

```diff
@@ -20,12 +20,8 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => {/* Fill me in */ }}>
-                    +
-                </button>
-                <button onClick={() => {/* Fill me in */ }}>
-                    -
-                </button>
+                <button onClick={() => this.props.increment() }>+</button>
+                <button onClick={() => this.props.decrement() }>-</button>
```

Comment:
Since you're not passing in any arguments, there's no need to use anonymous functions here (which will create an extra function every time a button is clicked). Instead, pass in the function without invoking it (example:: `onClick={this.props.someFunction}`).